### ISSUE TITLE
Add learning rate schedules

### ISSUE THREAD

**Issue by jeremiecoullon** [jeremiecoullon/SGMCMCJax#9]
Add learning rate schedules to the diffusions. Follow how Jax does it for [optimisers](https://jax.readthedocs.io/en/latest/_modules/jax/experimental/optimizers.html#constant)

**Comment by jeremiecoullon** [comment]
Added the constant and polynomial schedules.
